iphone/Classes/TiUIProgressBar.m

```diff
@@ -67,7 +67,10 @@ -(CGFloat)contentHeightForWidth:(CGFloat)width
 {
 	CGSize fontSize = [self sizeForFont:width];
 	CGSize progressSize = [progress sizeThatFits:fontSize];
-	return fontSize.height + progressSize.height;
+    if (messageLabel == nil) {
+        return fontSize.height + progressSize.height;
+    }
+	return fontSize.height + progressSize.height + 5;
```

Comment:
Indentation is incorrect. I also feel like this is not the proper way to fix it (by manually setting a constant value of 5 just to unclip it). I need to spend more time reviewing the progress-bar layout-code this week.